ai/STA/Strategy/prepare_kickoff_defense.py

```diff
@@ -30,18 +30,18 @@ def __init__(self, p_game_state):
             else -GameState().const["CENTER_CENTER_RADIUS"]
 
         # Positions objectifs des joueurs
-        attack_top_position = Pose(GameState().const["FIELD_OUR_GOAL_X_EXTERNAL"] / 10,
+        attack_top_position = Pose.from_values(GameState().const["FIELD_OUR_GOAL_X_EXTERNAL"] / 10,
```

Comment:
we should make a comment that this is hardcoded and bad. our robot will alway face at 0 degree